esmvaltool/utils/recipe_test_workflow/flow.cylc

```diff
@@ -1,21 +1,24 @@
 #!jinja2
+{{ assert(SITE != "", "SITE must be set to something other than an empty string") }}
+
+# Add recipes to the RTW in your site's `<site>-recipes.cylc` file in the 'site'
+# directory. See "How to add a recipe to the RTW" documentation for more information.
+{% set SITE_RECIPES_FILE = 'site/' ~ SITE ~ '-recipes.cylc' %}
+{% from SITE_RECIPES_FILE import FAST_RECIPES, MEDIUM_RECIPES %}
```

Comment:
I know you spent a lot of time investigating this, and apologies if you already told me the answer to this question, but if the file is included (e.g. `{% include 'site/' ~ SITE ~ '-recipes.cylc' %}`) are the `FAST_RECIPES` and `MEDIUM_RECIPES` variables not recognised? 🤔

Reply:
I asked at the Cylc Surgery about this and, even though I am unable to find any documentation about it, including a file processes any Jinja2 code in the file and includes the result, rather than including the raw code directly. This means any variables that are `set` in the Jinja2 code essentially disappear when the file is included. Importing (as has been done here) is an appropriate approach to retain the variables 👍